github/ci/prow-deploy/kustom/base/manifests/local/prow-test-subset-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-subset
+  labels:
+    app: test-subset
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prow-test-subset
+  template:
+    metadata:
+      labels:
+        app: prow-test-subset
+    spec:
+      serviceAccountName: prow-test-subset
+      terminationGracePeriodSeconds: 180
+      containers:
+        - name: test-subset
+          image: quay.io/kubevirtci/test-subset:v20240108-87253b59
+          imagePullPolicy: IfNotPresent
+          args:
+            - --github-token-path=/etc/github/oauth
+            - --github-endpoint=http://ghproxy
+            - --github-endpoint=https://api.github.com
+            - --cache-dir=/var/run/cache
+            - --jobs-config-base=github/ci/prow-deploy/files/jobs
+            - --prow-config-path=github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml
+            - --prow-location=https://raw.githubusercontent.com/kubevirt/project-infra/main
+          ports:
+            - name: http
+              containerPort: 9900
+          volumeMounts:
+            - name: hmac
+              mountPath: /etc/webhook
+              readOnly: true
+            - name: oauth
+              mountPath: /etc/github
+              readOnly: true
+            - name: plugins
+              mountPath: /etc/plugins
+              readOnly: true
+            - name: cache
+              mountPath: /var/run/cache
+              readOnly: false
+      volumes:
+        - name: hmac
+          secret:
+            secretName: hmac-token
+        - name: oauth
+          secret:
+            secretName: oauth-token
```

Comment:
atm the plugin doesn't comment, if we add comments or in case required we will need to change to `commenter-oauth-token` (according comparison to phased plugin)